.env.example

```
@@ -1,47 +1,71 @@
-# Email (SendGrid)
-SENDGRID_API_KEY=your_sendgrid_api_key
-SENDGRID_FROM_EMAIL=noreply@procrechesolutions.com
+# Session
+SESSION_SECRET=your_session_secret_min_32_chars
 
-# Feature Flags
-FEATURE_I18N_ENABLED=true
-FEATURE_CLAMAV_ENABLED=true
-FEATURE_GATED_CONTENT_ENABLED=true
+# API Rate Limiting
+RATE_LIMIT_TTL=60000  # 1 minute
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)